src/config/ingress/values.helm.yaml

```diff
@@ -1,12 +1,14 @@
 # NGINX Ingress Controller helm configuration
 # https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/values.yaml
-# Placesholders #..# will be replaced automatically
 
 controller:
+  securityContext:
+    readOnlyRootFilesystem: true # If a container does not need to write files, it should be run with a read-only filesystem. 
+    privileged: false # Running a container as privileged gives all capabilities to the container, and it also lifts all the limitations enforced by the device cgroup controller.
   autoscaling:
-    enabled:     true
-    minReplicas: 3
-    maxReplicas: 12 
+    enabled:     true # Enable autoscaling via horizontal pod autoscaler
+    minReplicas: 3 # Minimum number of nodes
```

Comment:
cant we add PDB to ngnix as well?

Reply:
the current helm chart doesn't allow that - might be worth a contribution - let me give it a try

Reply:
it's already there - by default - the no. of `minAvailable` replicas can be configured..

```console
kubectl get pdb -n kube-system
NAME                       MIN AVAILABLE   MAX UNAVAILABLE   ALLOWED DISRUPTIONS   AGE
coredns-pdb                1               N/A               1                     9d
ingress-nginx-controller   1               N/A               2                     9d
konnectivity-agent         1               N/A               1                     5d6h
```

Reply:
ah cool. Can you just mentioned that in the docs as well with a sentence?